aiohttp/_frozenlist.pyx

```diff
@@ -18,6 +18,9 @@ cdef class FrozenList:
         if self.frozen:
             raise RuntimeError("Cannot modify frozen list.")
 
+    cdef object _fast_len(self):
```

Comment:
The function is used only once, please inline it -- the speed should be the same.

Reply:
done